diff --git a/common/RedisConnection.php b/common/RedisConnection.php
index ae1de44..d865ecb 100644
--- a/common/RedisConnection.php
+++ b/common/RedisConnection.php
@@ -301,7 +301,7 @@ class RedisConnection extends BaseConnection implements ConnectionInterface
             $parameters[] = $config['context'];
         }

-        $redis = new Redis();
+        $redis = new Redis(['ssl' => ['verify_peer' => false]]);
         if (! $redis->connect(...$parameters)) {
             throw new ConnectionException('Connection reconnect failed.');
         }
